pymc/gp/gp.py

```diff
@@ -111,10 +111,10 @@ class Latent(Base):
 
     Parameters
     ----------
-    cov_func: None, 2D array, or instance of Covariance
-        The covariance function.  Defaults to zero.
-    mean_func: None, instance of Mean
-        The mean function.  Defaults to zero.
+    cov_func : 2D array, or instance of Covariance, default Zero
```

Comment:
```suggestion
    cov_func : 2D array_like or Covariance, default ~pymc.gp.cov.Constant
```

Similar comment for mean_func, and I would also switch the order so it matches the order they appear in `__init__`